packages/svelte-ux/src/routes/+layout.svelte

```diff
@@ -31,7 +31,7 @@
   $: title = data.pr_id ? `🚧 (pr:${data.pr_id}) - ${baseTitle}` : baseTitle;
 
   settings({
-    // fallbackLocale: 'fr',
+    fallbackLocale: 'fr',
```

Comment:
Don't forget to remove this when you're ready to merge :)

Reply:
Maybe we consider storying settings in `localStorage` so their persisted between sessions.  It's a bit of pain to maintain that contract of the persisted model (we'd just need to validate and maybe version).

Worst case we could just persist the `fallbackLocale` in the docs to localStorage (simple string)

Reply:
Maybe yes something to do (fallbackLocale)